banzai/logs.py

```diff
@@ -5,7 +5,7 @@
 
 
 class BanzaiLogger(logging.getLoggerClass()):
-    def __init__(self, name, level='NOTSET'):
+    def __init__(self, name, level='DEBUG'):
```

Comment:
This should be kept as `NOTSET`. I think I initially set it that way just to be consistent with the Logger super class. However, the above change makes the logging go a bit crazy, all of the imported modules start logging every detail and it's a bit of a mess.

Reply:
Fixed.